src/Route.php

```diff
@@ -186,13 +186,27 @@ public function pattern(string $pattern): self
     /**
      * @return self
      */
-    public function host(string $host): self
+    public function host(string ...$hosts): self
```

Comment:
That looks a bit weird in routes:

```php
->host('demo.test', 'demo.com')
```

According to English grammar it should be hosts, not host. Maybe we need an alias method?

```suggestion
    public function hosts(string ...$hosts): self
    {
        // call host() here
    }

    public function host(string ...$hosts): self
```